.mailmap

```diff
@@ -7,3 +7,4 @@ Gisle Aas <gaas@cpan.org> <gisle@activestate.com>
 Gisle Aas <gaas@cpan.org> <gisle@baux-2.lan>
 Gisle Aas <gaas@cpan.org> gisle <gisle>
 Gisle Aas <gaas@cpan.org> aas <aas>
+Nicolas R <nicolas@atoomic.org> <atoomic@cpan.org>
```

Comment:
I've added https://github.com/Dual-Life/mime-base64/pull/7 to provide this
but we can cancel it if this one merge earlier :-)